Add .dockstore.yml (#139)
common-workflow-library/bio-cwl-tools: CONTRIBUTING.md

While there is no specific standard on the order of sections in a CWL file, [cwl-format](https://github.com/rabix/cwl-format) may be used to format the CWL
code of a tool if the author feels that it is useful.

## Dockstore

> Update [.dockstore.yml](.dockstore.yml) to allow your tool to be automatically registered on [Dockstore](https://dockstore.org/).  If your tool's primary CWL path is `/foo/foo-bar.cwl`, you would add the following entry to the `tools` section of [.dockstore.yml](.dockstore.yml):

Reviewer: ```suggestion
Update [.dockstore.yml](.dockstore.yml) to allow the new tool description to be automatically registered on [Dockstore](https://dockstore.org/).
If the path within this repository to the tool's description is `/foo/foo-bar.cwl`, you would add the following entry to the `tools` section of [.dockstore.yml](.dockstore.yml):
```
